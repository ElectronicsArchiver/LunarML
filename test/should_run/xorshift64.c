#include <stdio.h>
#include <inttypes.h>

uint64_t seed = 0xBADCAFE;
uint64_t xorshift64(void)
{
    seed ^= seed << 13;
    seed ^= seed >> 7;
    return seed ^= seed << 17;
}

int main(void)
{
    for (int i = 0; i < 100; ++i) {
        printf("0x%" PRIX64 "\n", xorshift64());
    }
}
