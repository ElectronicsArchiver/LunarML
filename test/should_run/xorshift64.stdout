0x2EEB2588441EEEB
0x626D84A7703B7F6
0x4F9B627A4FCC8D19
0x243A30723D557243
0x5C9E5B4172F52867
0xA53249599FB83BF7
0x7D76C3B569775640
0x78C70736E56028EC
0xA94A335772C753BD
0xD79593B6A335925A
0x6797E3E7EEA02FFE
0xAC3D11D2CCB5021
0x18DB28E893C0EEC1
0x10F6E8EC51BCFF5C
0xA44C4789DA8FD1A2
0x20A727D45DF8E681
0xEBB23866B9B2970C
0x88FB078528C1B122
0x826BFC6851583AC0
0x8B429AF7A5C63AB5
0x748BAE388B55BB80
0x4D17169779B3F0F7
0x4A3EF916D00E4AD6
0xA294759D70E023C3
0x41378D490F97344
0x99E3AAC788A8D0A2
0xB7A84256E49FE983
0x96C7CC13EBCD690
0xE0B8A1EC9F5C0B3D
0x85AA186CA8D1DC6B
0x43F23E98745C0513
0xE0CED45CE0A599D9
0xBF8044ABCA6D84AA
0x99407B87DC4B3523
0x5501AE3BB0888B89
0xCB32BCEE450658DE
0xC8DA0CA554DFA3EF
0xFEB980B4D3330768
0x46FC69946075BB66
0x3AFABD3F71974990
0xBED0EE8385920303
0x9C5DD3FEA9F387C5
0xA3F4F3BF3BAD318A
0x183E03AD9A754969
0x96FECDFF4648D9BB
0x127C4F6BC5546C8
0x3AFFB810502F5E45
0x5E856851F3E231B9
0x636B95FC998ABB9A
0x4C77040845BE086D
0x6CE217D365C0CF3D
0xB97F1E23E91921E3
0x71838CB8DA7F0B60
0x7DA514128F892D76
0xC849D18F282BA2AC
0x7A5A556EF514DEE9
0xF92188F32C3C6D14
0xECF30315E941D0CE
0xB70BCCBCE920A0EF
0x1D76F0B5B81B3A6E
0xCC95E2E91DDC579A
0xD18934D2A16B49B5
0x7A8129A493015266
0xFE6CB0CD22BB0942
0x95C3109BA8B46FD0
0x7CB3BB4FD91AF30F
0xE21548079427E529
0x686F4D51EF15C0A3
0x7462990D766BA3E2
0x4452B5881A59CC25
0xF19B3A9B0860D6FD
0x1A5B2142CBA0810
0x45502F2A9B637800
0x6D48E5A9046BBEF0
0xC42480B9048D58D
0x3D24A3AF81208766
0x4A5AFCF3393FDFE8
0x3029C46DDFE95A57
0x666A9DFA2F0CFD23
0xD2E97C381FFBCC19
0xBAE57250D3361DC1
0x794ACFB83FDB21BA
0x475B0964AFA9B979
0xC16A04C79321944B
0xA66CDE99ADADA5A3
0x143016ADAD7BF628
0xC641C477C1918BC4
0xBB48542CC8AED9D3
0x1DE895C440F390A0
0x748DB0E539805F81
0xA8E105D3A6E89F7E
0x69297360BED51C0
0x2912D3CAC050FB63
0xCA23E1D29A28E255
0xB490D32FA4CC86D1
0x345A7D044F468B9C
0x79078F17841F618B
0xE3223497FBEE5D88
0xF00062022259E333
0xB1B3BE027469FC35
